editoast/src/views/train_schedule/mod.rs

```diff
@@ -348,7 +350,14 @@ async fn standalone_simulation(
     // Start a transaction
     conn.transaction::<_, InternalError, _>(|conn| {
         // Save inputs
-        for train_schedule in train_schedules {
+        for mut train_schedule in train_schedules {
+            train_schedule.infra_version = infra.version.clone();
+            train_schedule.rollingstock_version = lightrollingstock
+                .results
+                .iter()
+                .filter(|r| (r.rolling_stock.id == train_schedule.rolling_stock_id))
+                .map(|r| r.rolling_stock.version)
+                .next();
```

Comment:
Why not call `LightRollingStock.retrieve()` instead of iterating on the list of rolling stocks ?